pillar.example

```diff
@@ -1,6 +1,12 @@
 vault:
-  lookup:
-    pkg: myvault
-    conf_file: /usr/local/etc/vault/vault.conf
+  overrides:
+    keybase_users:
+      - renaissancedev
+      - pdpinch
+      - bdero
+    secret_shares: 3
+    secret_threshold: 2
   config:
     key: othervalue
```

Comment:
It doesn't look like this "config" key in the pillar is referenced. Is it being used?